website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+---
+title: The State the React Native Community in 2018
+author: Lorenzo Sciandra
+authorTitle: Core Maintainer & React Native Developer
+authorURL: https://github.com/kelset
+authorImageURL: https://avatars2.githubusercontent.com/u/16104054?s=460&v=4
+authorTwitter: kelset
+category: announcements
+---
+
+Two years from now, we’ll probably look back at 2018 as a turning point in the React Native development history.
+
+A lot of eyes are currently pointed at the incoming rewrite of the architecture, widely known as [Fabric](https://github.com/react-native-community/discussions-and-proposals/issues/4), which will enable our mobile apps to reach new peaks of mobile performances (alongside JSI & TurboModules, which are equally as important to reach said goal).
+
+But I will argue that the biggest shift initiated during 2018 is not code related;  this title should go to the empowerment that the React Native Community gained.
+
+Since the very beginning, Facebook allowed developers from all around the world to participate in this Open Source project; investing a few super active members of the community of responsibilities like owning the release cycle.
+
+And, this year, the members of the Core Contributors group took a few substantial steps towards making the whole community more able to shape the future of this awesome project: two repositories have been created throughout the year precisely with this goal in mind.
+
+## [`react-native-releases`](https://github.com/react-native-community/react-native-releases) 📬
+
+This repository, started around Feb, served the dual purpose of allowing everyone to keep up the new releases in a more collaborative manner and opened the conversation of what would be part of a certain release to whoever wanted to suggest a cherry-pick (like for [0.57.8](https://github.com/react-native-community/react-native-releases/issues/71) and all its previous versions).
+
+This has been the driving force behind moving away from a monthly release cycle, and the "long term support" approach currently used for version 0.57.x.
+
+Half of the merit in reaching these decisions goes to the other repository created this year:
+
+## [`discussions-and-proposals`](https://github.com/react-native-community/discussions-and-proposals) 🗣
+
+This other ‘meta’ repository, created a few months after its sister (in July), expanded on the idea of a more open environment for conversations on React Native; while, partially, this need was covered by a few issues labelled [`For Discussion`](https://github.com/facebook/react-native/labels/For%20Discussion) in the main repository, we wanted to give our own spin to the RFC approach that other libraries (like React) have.
```

Comment:
I would drop the "sister" thing. Just say "This repository was created in July"

Reply:
👌